annotation-solr/src/main/java/eu/europeana/annotation/solr/service/impl/SolrAnnotationServiceImpl.java

```diff
@@ -539,18 +539,19 @@ else if(BodyInternalTypes.isSimpleTagBody(anno.getBody().getInternalType())) {
 	}
 	
 	private SolrQuery solrUniquenessQueryTranscriptions(Annotation anno, boolean noSelfDupplicate) {
-	  SolrQuery query = new SolrQuery();
-	  query.setQuery(SolrAnnotationConstants.TARGET_URI + ":\"" + anno.getTarget().getSource() + "\"");
-	  query.addFilterQuery(WebAnnotationModelFields.MOTIVATION + ":\"" + MotivationTypes.TRANSCRIBING.getOaType() + "\"");
-	  query.addFilterQuery(SolrAnnotationConstants.BODY_VALUE_PREFIX + anno.getBody().getLanguage() + ":*");
+	  SolrQuery query = new SolrQuery();	
+      query.setQuery(WebAnnotationModelFields.MOTIVATION + ":\"" + MotivationTypes.TRANSCRIBING.getOaType() + "\"");
+      query.addFilterQuery(SolrAnnotationConstants.TARGET_URI + ":\"" + anno.getTarget().getSource() + "\"");
```

Comment:
actually I think that the scope should be unique. For this purpose, we should probably use the target_record_id instead of target URIs, but we need to agree this with Hugo